evap/evaluation/tests/test_models.py

```diff
@@ -116,6 +117,39 @@ def test_responsible_contributors_ordering(self):
         course = Course.objects.get(pk=course.pk)
         self.assertEqual(list(course.responsible_contributors), [responsible2, responsible1])
 
+    def test_single_result_can_be_deleted_only_in_reviewed(self):
+        responsible = mommy.make(UserProfile)
+        course_type = mommy.make(CourseType)
+        course = Course(semester=mommy.make(Semester))
+        form_data = {
+            "name_de": "qwertz",
+            "name_en": "qwertz",
+            "type": course_type.pk,
+            "degrees": ["1"],
+            "event_date": "02/1/2014",
+            "responsible": responsible.pk,
+            "answer_1": 6,
+            "answer_2": 0,
+            "answer_3": 2,
+            "answer_4": 0,
+            "answer_5": 2,
+            "semester": course.semester.pk
+        }
+        form = SingleResultForm(form_data, instance=course)
+        self.assertTrue(form.is_valid())
+        form.save(user=mommy.make(UserProfile))
+
+        self.assertTrue(Course.objects.filter(pk=course.pk).exists())
```

Comment:
can you replace all that by directly creating a singleResult with model mommy? this test doesn't need to use a form at all as far as i can see.